src/disco/css/Addon.scss

```diff
@@ -120,4 +92,40 @@ $addon-padding: 20px;
     }
   }
 
+  .notification {
+    align-content: stretch;
+    align-items: center;
+    bottom: 0;
+    color: #fff;
+    display: flex;
+    flex-direction: row;
+    left: 0;
+    padding: 15px 20px;
+    position: absolute;
+    right: 0;
+    top: 0;
+
+    &.error {
+      background: #c33c32 url('../img/warning.svg') no-repeat calc(100% - 40px) 50%;
+      z-index: 20;
```

Comment:
Why is the error higher than needs restart? So you see an error if it is in `ERROR` and `needsRestart`?